src/pages/NewBeer.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState, useEffect } from 'react'
+import { Link, useNavigate  } from 'react-router-dom';
+import axios from 'axios';
+
+function NewBeer() {
+    const [name, setName] = useState('');
+    const [tagline, setTagline] = useState('');
+    const [description, setDescription] = useState('');
+    const [firstBrewed, setFirstBrewed] = useState('');
+    const [brewersTips, setBrewersTips] = useState('');
+    const [attenuationLevel, setAttenuationlevel] = useState(0);
+    const [contributedBy, setContributedBy] = useState('');
+
+    const handleName = (e) => setName(e.target.value);
+    const handleTagline = (e) => setTagline(e.target.value);
+    const handleDescription = (e) => setDescription(e.target.value);
+    const handleFirstBrewed = (e) => setFirstBrewed(e.target.value);
+    const handleBrewersTips = (e) => setBrewersTips(e.target.value);
+    const handleAttenuationlevel = (e) => setAttenuationlevel(e.target.value);
+    const handleContributedBy = (e) => setContributedBy(e.target.value);
+
+    const navigate = useNavigate();
+
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        try {
+            const body = { name, tagline, description, firstBrewed, brewersTips, attenuationLevel, contributedBy};
```

Comment:
Since the tags that the api accepts for the beer are different from the ones that you named, here you need to do something like this:
`const body = { name, tagline, description, first_brewed: firstBrewed, brewers_tips: brewersTips, attenuation_level: attenuationLevel, contributed_by: contributedBy }`
This is why when you created a beer the firstBrewed, brewersTips, attenuationLevel and contributedBy were not working.